packages/react-core/src/utils/geo.js

```diff
@@ -1,12 +1,16 @@
+import bboxClip from '@turf/bbox-clip';
 import bboxPolygon from '@turf/bbox-polygon';
+import union from '@turf/union';
+import { getType } from '@turf/invariant';
+import { polygon, multiPolygon } from '@turf/helpers';
```

Comment:
Ok, it sounds like we should add those as peerDependencies in react-core, right?

Reply:
Don't we have them already?